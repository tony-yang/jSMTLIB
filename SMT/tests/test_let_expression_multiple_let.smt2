(set-logic QF_LIA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status unsat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Int)
(declare-fun x2 () Int)
(declare-fun x3 () Int)
(declare-fun x4 () Int)
(declare-fun x5 () Int)
(assert (let ((?v_0 (+ (* 3 x1) (- x2))) (?v_2 (= x3 6))) (let ((?v_1 (- x4 x5))) (and (> ?v_0 0) (= ?v_1 10) ?v_2))))
(check-sat)
(exit)
