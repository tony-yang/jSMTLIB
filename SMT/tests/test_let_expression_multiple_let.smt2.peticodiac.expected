# set_logic: QF_LIA
# :source: | SMT-COMP'06 organizers |
# :smt-lib-version: 2.0
# :category: "check"
# :status: unsat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 5 3
c 0.0 0.0 1.0 0.0 0.0
b 5 =:6.0 =:6.0
c 3.0 -1.0 0.0 0.0 0.0
b 6 >:0.0 NO_BOUND
c 0.0 0.0 0.0 1.0 -1.0
b 7 =:10.0 =:10.0
